composer/loggers/mlflow_logger.py

```diff
@@ -36,16 +37,16 @@
 DEFAULT_MLFLOW_EXPERIMENT_NAME = 'my-mlflow-experiment'
 LOG_DUPLICATED_METRIC_VALUE_PER_N_STEPS = 100
 
+spawn_context = multiprocessing.get_context('spawn')
 
-class MlflowMonitorProcess(multiprocessing.Process):
+
+class MlflowMonitorProcess(spawn_context.Process):
```

Comment:
nit: just a little cleaner so you can omit the first line
```suggestion
class MlflowMonitorProcess(multiprocessing.context.SpawnProcess):
```

Reply:
discussed offline, I prefer my way, but neither of us care that much, so just going to leave it